tests/Fixtures/e2e/PSR_0_Autoloader/src/PSR_0_Autoloader/SourceClass.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace PSR_0_Autoloader;
+
+class SourceClass
```

Comment:
That's just as same as the current PSR-4. I think if test best to test a really old-style `Namespace_Class_Name` style project. 

For example, I could not run Infection on [pear/OLE](https://github.com/pear/OLE). It gives:
```
Processing source code files: 4/7
In CodeCoverageClassIgnoreVisitor.php line 63:
                                 
  Class OLE_Test does not exist  
```

On subsequent, for some very interesting reason:
```
In JsonValidationException.php line 51:
                                                                                            
  "OLE/infection.json.dist" does not match the expected JSON schema:  
   - source.excludes : Object value found, but an array is required                         
```
The last error possibly related to these configuration steps:
```
Any directories to exclude from within your source directories?: tests
Any directories to exclude from within your source directories?: coverage
Any directories to exclude from within your source directories?: build
Any directories to exclude from within your source directories?: vendor
Any directories to exclude from within your source directories?: 
```
infection.json has this:
```
 "excludes": {
     "0": "vendor",
     "1": "tests",
     "3": "coverage",
     "4": "build"
 }
```

Reply:
Actually, that's not an unknown issue with "Class OLE_Test does not exist", just #576